.github/workflows/ci.yml

```
@@ -16,32 +16,16 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-22.04, macos-14]
-        java-version: [11]
         ballerina-version: [2201.5.0]
 
     steps:
       - name: Checkout code
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
 
-      - name: Setup Java
-        uses: actions/setup-java@8df1039502a15bceb9433410b1a100fbe190c53b
+      - uses: ballerina-platform/setup-ballerina@7f21a19064107a8c9624a9d552e408965f0835a9
```

Shall we use the V1 here?
```suggestion
      - uses: ballerina-platform/setup-ballerina@v1
```

---

Exercism prefers to pin to a specific SHA rather than a tag.